Configuration/TypoScript/Library/lib.menu.main.setupts

```diff
@@ -1,9 +1,10 @@
-lib.menu.main = HMENU
-lib.menu.main {
+lib.menu.main = COA
+lib.menu.main.100 = HMENU
+lib.menu.main.100 {
     entryLevel = 0
     includeNotInMenu = 0
     excludeUidList = {$themes.configuration.menu.main.excludeUidList}
-    # wrap = <ul class="main-navigation__items-list js__main-navigation__items-list"> | </ul>
+    #wrap = <ul class="main-navigation__items-list js__main-navigation__items-list">|</ul>
```

Comment:
this wrapper is added in Header partial. it was not comment by me, looks like i only removed some spaces